pkg/nodes/enricher/metrics.go

```diff
@@ -34,10 +40,17 @@ func newMetrics(subsystem pkgMetrics.Subsystem) metrics {
 			Help:      "Amount of time it's taken to scan a node in ms",
 			Buckets:   prometheus.ExponentialBuckets(4, 2, 16),
 		}, []string{"Scanner", "Error"}),
+		nodeInventoryComponentSize: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: pkgMetrics.PrometheusNamespace,
+			Subsystem: subsystem.String(),
+			Name:      "node_scan_num_components",
+			Help:      "Number of discovered components per Node",
```

Comment:
per node? it looks like it's per cluster (I think)

Reply:
We call it in the `pkg/nodes/enricher/enricher_impl.go`, so it is per node as this pipeline gets separate message for each node in the cluster.

Reply:
We want to track this per node, so I added another label (`Node`) to the metric in [24b2d80](https://github.com/stackrox/stackrox/pull/8686/commits/24b2d80aa9aff596a7a9501af6bd24ea26f213d5)